app/views/templates/search_filter_template.scala.html

```diff
@@ -28,6 +28,7 @@ <h2 class="filter-head govuk-body govuk-!-font-size-24">
 
```

Comment:
L25 to L27 (above) need removing and any formatting errors need correcting in the custom CSS

Reply:
Why remove completely? don't we need the filter.heading ?

Reply:
No, the heading duplicates the content in the legend. Removing the heading was a specific point raised in the accessibility audit